modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/treeview/NodeList.js

```diff
@@ -44,7 +44,12 @@ export default function NodeList({
 			}}
 			onFocus={(event) => {
 				if (onFocus) {
-					onFocus(event);
+					onFocus(event, {useKeyboardNavigation: true});
```

Comment:
`use` prefix in React world implies a hook. This could be, for example `keyboardNavigation`